modules/ROOT/pages/organization-management-in-bonita-bpm-studio.adoc

```diff
@@ -29,7 +29,7 @@ To specify an organization manually, you create the organization, create the gro
 . Go to the *_Group_* tab to add groups to your organization:
  ** Click *_Add_* to add a group. You can specify the following:
   *** the group Name, which is used internally by the Bonita software (*NB:* Bonita doesn't support the '/' character in the group name field)
```

Comment:
group name field. => group name field. It is used as path separator. )

Reply:
Done